File: 5 kyu/Sum_of_Pairs.py
def sum_pairs(ints, s):
    o = {}
    for i in range(len(ints)):
        temp = ints[i]
        temp2 = s - temp
        if temp2 in o:
            return [temp2, temp]
        else:
            o[temp] = 1
    return None       

File: 5 kyu/Sum of Pairs.js
var sum_pairs=function(ints, s){
     var obj = {};
     for(let x = 0; x < ints.length; x++) {
         let temp = ints[x];
         let temp2 = s - temp;
         if(temp2 in obj) {
             return [temp2, temp];
         } else {
             obj[temp] = 1;
         }
     }
     return undefined;
}
